pkg/registry/batch/job/storage/storage.go

```diff
@@ -51,6 +52,9 @@ func NewStorage(optsGetter generic.RESTOptionsGetter) (JobStorage, error) {
 	}, nil
 }
 
+var deleteOptionWarnings = "Set propagationPolicy=Orphan, if you want to intentionally orphan pods. If you want to " +
+	"clean up children, you should set propagationPolicy=Background."
+
```

Comment:
I'd suggest something more succinct like `child pods are preserved by default when jobs are deleted; set propagationPolicy=Background to remove them or set propagationPolicy=Orphan to suppress this warning`

Reply:
Done. Thanks for the quick review